gomint-server/src/main/java/io/gomint/server/entity/monster/EntityPhantom.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) 2020 Gomint team
+ *
+ * This code is licensed under the BSD license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package io.gomint.server.entity.monster;
+
+import io.gomint.enchant.Enchantment;
+import io.gomint.enchant.EnchantmentLooting;
+import io.gomint.entity.Entity;
+import io.gomint.entity.passive.EntityHuman;
+import io.gomint.inventory.item.ItemPhantomMembrane;
+import io.gomint.inventory.item.ItemStack;
+import io.gomint.math.Location;
+import io.gomint.server.entity.Attribute;
+import io.gomint.server.entity.EntityLiving;
+import io.gomint.server.entity.EntityTags;
+import io.gomint.server.entity.EntityType;
+import io.gomint.server.registry.RegisterInfo;
+import io.gomint.server.world.WorldAdapter;
+
+import java.util.Set;
+import java.util.concurrent.ThreadLocalRandom;
+
+/**
+ * @author joserobjr
+ * @since 2021-01-12
+ */
+@RegisterInfo( sId = "minecraft:phantom" )
+public class EntityPhantom extends EntityLiving implements io.gomint.entity.monster.EntityPhantom {
+
+    /**
+     * Constructs a new EntityPhantom
+     *
+     * @param world The world in which this entity is in
+     */
+    public EntityPhantom( WorldAdapter world ) {
+        super( EntityType.PHANTOM, world );
+        this.initEntity();
+    }
+
+    /**
+     * Create new entity phantom for API
+     */
+    public EntityPhantom() {
+        super( EntityType.PHANTOM, null );
+        this.initEntity();
+    }
+
+    private void initEntity() {
+        this.setSize( 0.9f, 0.5f );
+        this.addAttribute( Attribute.HEALTH );
+        this.setMaxHealth( 20 );
+        this.setHealth( 20 );
+    }
+
+    @Override
+    protected void kill() {
+        super.kill();
+
+        if (isDead()) {
+            return;
+        }
+        
+        if (!isLastDamageCausedByPlayer()) {
+            return;
+        }
+
+        Location location = this.getLocation();
+
+        // Drop items
+        Entity lastDamageEntity = this.lastDamageEntity;
+        int looting = 0;
+        if (lastDamageEntity instanceof EntityHuman) {
+            Enchantment enchantment = ((EntityHuman) lastDamageEntity).getInventory().getItemInHand().getEnchantment(EnchantmentLooting.class);
+            if (enchantment != null) {
+                looting = enchantment.getLevel();
+            }
+        }
+
+        int amount = ThreadLocalRandom.current().nextInt(3 + Math.min(looting, 3));
```

Comment:
Why is this looting level capped, whilst the others aren't? For example see EntityPanda#79 or EntityDolphin#81

Reply:
Because the wiki said `for a maximum of 5 with Looting III`, the panda page didn't, but I missed that on dolphin which also mentions a limit.

Do you prefer with or without a level cap?